src/Quasar/Mount/SparkHDFS.purs

```diff
@@ -54,7 +54,7 @@ fromJSON ∷ Json → Either String Config
 fromJSON
   = fromString
   <=< (_ .? "connectionUri")
-  <=< (_ .? "spark")
```

Comment:
I think we should open another issue for this anyway, it's not relevant to the code changes in this PR.